### ISSUE TITLE
Upgrade to Java 13

### ISSUE THREAD

**Issue by nizap** [Mari-Wie/ArchCNL#7]

**Comment by vhschlenker** [comment]
Maybe use a LTS Version of JDK as target, like 11.
Reference: [Oracle Java SE Support Roadmap](https://www.oracle.com/java/technologies/java-se-support-roadmap.html)

**Comment by Mari-Wie** [comment]
Good idea, let's decide it when the topic is adressed. There may be a new LTE released (09/2021).

**Comment by nizap** [comment]
The Java version has been switched to Java 11 by 
53812d2.

**Comment by Mari-Wie** [comment]
Will you close the issue then? Or why is it still open?
